locale/af.po

```diff
@@ -1335,15 +1335,9 @@ msgstr ""
 msgid "DIFFICULTY_PRESET_NORMAL"
 msgstr ""
 
-msgid "DIGESTION_EFFICIENCY"
```

Comment:
Seems like some stuff is not handled by the new tooltips yet.

Reply:
Isn't it added back some lines later?

Reply:
Lines that start like:
```
#~ msgid "DIGESTION_EFFICIENCY"
#~ msgstr "Digestion Efficiency"
```

Are *comments*, so the translation has been removed. But gettext keeps old translations like that as comments so if in the future the key is added back, it will restore the translation. But as things are it is currently in deleted state.